Docs/source/building/juwels.rst

```diff
@@ -0,0 +1,64 @@
+.. _building-juwels:
+
+Juwels (JSC)
+============
+
+The `Juwels supercomputer <https://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JUWELS/JUWELS_node.html>`_ is located at JSC.
+
+See `this page <https://apps.fz-juelich.de/jsc/hps/juwels/quickintro.html>`__ for a quick introduction.
+
+Installation
+------------
+
+We use the following modules and environments on the system.
```

Comment:
I do a minimal reordering of sections in #1127 - can you please apply the same order here, too? :)